dali/python/nvidia/dali/pipeline.py

```diff
@@ -478,7 +478,7 @@ def build(self, define_graph = None):
         self._pipe.Build(self._names_and_devices)
         self._built = True
 
-    def feed_input(self, data_node, data, layout="", cuda_stream = None):
+    def feed_input(self, data_node, data, layout="", cuda_stream = None, use_copy_kernel = False):
```

Comment:
You need to extend external_source.py as well.

Reply:
done